### PR TITLE
Remove unused key-value pairs from bucket consts

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

We had been importing the `DEVELOPMENT`, `PRODUCTION`, and `STAGING` values from the `environments` file and using them as keys to set bucket values, but those values were not being exported from the environments file.  Having keys like `STAGING` and `VAGOVSTAGING` only adds confusion, so this PR removes the keys that are no longer used.

The exported values:

https://github.com/department-of-veterans-affairs/vets-website/blob/5836f211f0033506ff3c07836d39a7612cc9d1e2/src/site/constants/environments.js#L6-L18

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/constants/buckets.js
Comment (U-DON) [inline src/site/constants/buckets.js:4]: Are these useful in any other context? Seems confusing to mix these and the `VAGOV*` variants.

**Review by U-DON — APPROVED** [review]

File: src/site/constants/buckets.js
Comment (bkjohnson) [reply src/site/constants/buckets.js:4]: Confusing is right. There was a [recent devops PR](https://github.com/department-of-veterans-affairs/devops/pull/7296) that prompted me to do this after my own confusion trying to understand what was going on with the s3 buckets.

To my knowledge they aren't useful in any other context.  I'll let @rianfowler take a look before I merge though.

Comment (ncksllvn) [reply src/site/constants/buckets.js:4]: They were the vets.gov environments - dev|staging|www.vets.gov - and for a period they coexisted with the va.gov environments.

**Review by ncksllvn — APPROVED** [review]

**Pull request merged**
